dashboard/app/controllers/code_review_notes_controller.rb

```diff
@@ -14,18 +16,23 @@ def create
       }
     )
 
-    # TODO: fetch project and authorize
-    # check that user is authorized to leave a comment
-    # check that code review exists and is open
-
     # We wait to authorize until this point because we need to know
     # who owns the project that the comment is associated with.
-    # authorize! :create, @code_review_note, @project_owner, @project_id
+    authorize! :create, @code_review_note
 
     if @code_review_note.save
       return render json: @code_review_note.summarize
     else
       return head :bad_request
     end
   end
+
+  # PATCH /code_review_notes/:id/toggle_resolved
```

Comment:
(Closed the PR because it's a duplicate of https://github.com/code-dot-org/code-dot-org/pull/46424)

Got it. I'll change the route, I just copied the toggle resolved endpoint from v1, but this makes more sense.